### ISSUE TITLE
`Rancher Desktop` is not particularly friendly to Voice Over users

### ISSUE THREAD

**Issue by jsoref** [rancher-sandbox/rancher-desktop#2383]
### Actual Behavior

* horizontal splitters are pretty useless...
* groups are read by reading their entirety which is problematic
* the slider for member behaved pretty strangely at the end...
* while interacting with the slider, the highlight box was for larger than memory+cpus -- I don't know why, but I don't like it. -- it should have been just the memory box...
* items that trigger checkboxes/radio items don't clearly indicate that they will do so

### Steps to Reproduce

1. Enable Voice Over (with basic defaults)
2. Open Rancher Desktop
3. Interact w/ General...Kubernetes Settings (I got interrupted and stopped here)

### Result

* horizontal splitters:
    ![image](https://user-images.githubusercontent.com/2119212/173419992-32c440d3-a1e1-4450-80ee-8015db3a4ea6.png)
* groups:
    ![image](https://user-images.githubusercontent.com/2119212/173419897-57238957-d993-415e-abbe-b7e586c3e40b.png)
* sliders (really incomplete set):
    ![image](https://user-images.githubusercontent.com/2119212/173420306-6dc84736-15b0-41e5-92f6-e838240ceb33.png)
    ![image](https://user-images.githubusercontent.com/2119212/173420384-7e570d99-5ac4-4d0d-ba28-1e49db85d62f.png)
    ![image](https://user-images.githubusercontent.com/2119212/173420417-c15d2b8d-ef91-48bd-8ab2-b04be2dd8b37.png)
    ![image](https://user-images.githubusercontent.com/2119212/173420451-25093d72-7f36-46cf-b58e-8dcdd414ac1c.png)

### Expected Behavior

* probably don't mention horizontal splitters?
* read much less for groups?
* slider should be less frustrating?
* items that trigger checkboxes/radio items should indicate they're related to checkboxes

### Rancher Desktop Version

1.4.1

### Rancher Desktop K8s Version

(Off)

### Which container runtime are you using?

moby (docker cli)

### What operating system are you using?

macOS

### Operating System / Build Version

Monterey 12.4

### What CPU architecture are you using?

arm64 (Apple Silicon)

**Comment by jandubois** [comment]
Video on [Dropbox](https://www.dropbox.com/s/4oxuqf5v4q3l22v/Voice%20Over%20reading%20Rancher%20Desktop%20General..Kubernetes%20Settings.mov?dl=0)
